libmattermost.c

```diff
@@ -2135,6 +2156,7 @@ mm_roomlist_get_list(PurpleConnection *pc)
 		// Get a list of channels the user has already joined
 		mmtrl = g_new0(MatterMostTeamRoomlist, 1);
 		mmtrl->team_id = g_strdup(team_id);
+		mmtrl->team_desc = g_strdup(_("[joined channels]"));

```

Comment:
I don't know of any other place in Pidgin/libpurple that uses square brackets around a term like this.  Maybe just use `_("Joined channels")`